deploy/charts/grafana-multi-tenant-operator/crds/crd-grafana-datasource.yaml

```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: datasources.grafana.k8spin.cloud
+spec:
+  scope: Namespaced
+  group: grafana.k8spin.cloud
+  versions:
+    - name: v1
+      served: true
+      storage: true
+      schema:
+        openAPIV3Schema:
+          type: object
+          properties:
+            apiVersion:
+              description: 'APIVersion defines the versioned schema of this representation
+                of an object. Servers should convert recognized schemas to the latest
+                internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+              type: string
+            kind:
+              description: 'Kind is a string value representing the REST resource this
+                object represents. Servers may infer this from the endpoint the client
+                submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+              type: string
+            status:
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
+            spec:
+              type: object
+              required: [datasource, organizations]
+              properties:
+                datasource:
+                  type: object
+                  description: Grafana datasource in JSON format
+                  required: [name, data]
+                  properties:
+                    name:
+                      type: string
+                    data:
+                      type: string
+                organizations:
+                  description: Grafana Organizations the dashboard belongs to
+                  type: array
+                  items:
+                    type: string
+      additionalPrinterColumns:
+      - jsonPath: .spec.organizations
+        name: Organizations
+        type: string
+  names:
+    kind: Datasource
+    plural: datasources
+    singular: datasource
+    shortNames:
+      - dts
```

Comment:
Please, be consistent in the file formating, add a new line a the end of the file

Reply:
done